tests/test_taxonomy.py

```diff
@@ -0,0 +1,125 @@
+import re
+
+import pytest
+
+from openfoodfacts.taxonomy import (
+    Taxonomy,
+    TaxonomyNode,
+    create_taxonomy_mapping,
+    map_to_canonical_id,
+)
+
+
+def test_map_to_canonical_id():
+    taxonomy_mapping = {
+        "en:apple": "en:apples",
+        "en:apples": "en:apples",
+        "fr:pomme": "en:apples",
+        "fr:noix-d-isere": "en:nuts-from-isere",
+    }
+    values = [
+        "en: Apple",
+        "en: apples",
+        "fr: Pomme",
+        "fr: Bananes d'Isère",
```

Comment:
lol not many bananas around here 😂

Reply:
I needed a `d'` to check that taggification was done correctly, and "d'Isère" was the only think that came to my mind :laughing: